### ISSUE TITLE
Delete "secret.js"

### ISSUE THREAD

**Issue by gavinceldridge** [danielyaa5/therm-nest-cli#2]
You have all of your API credentials public! This is not good. Use a .gitignore file and get that info out of this project.

**Comment by danielyaa5** [comment]
Yes thanks, we need to switch to https://www.npmjs.com/package/configstore
